platform-tools-sdk/cargo-build-sbf/src/main.rs

```diff
@@ -614,13 +629,19 @@ fn build_solana_package(
 
         match cdylib_targets.len() {
             0 => {
-                warn!(
-                    "Note: {} crate does not contain a cdylib target",
+                info!(
+                    "Note: {} crate is a library to be imported by other solana programs",
                     package.name
                 );
                 None
             }
-            1 => Some(cdylib_targets[0].replace('-', "_")),
+            1 => {
+                info!(
+                    "Note: {} crate is a solana program ready for deployment",
+                    package.name
+                );
```

Comment:
I'm worried this message is noisy to print for every program build and doesn't provide a lot of extra information. I'd lean towards removing it, but I might be missing the point.

Reply:
Removed in https://github.com/anza-xyz/agave/pull/5729/commits/24284ddacc1bd1fee9a4df337d8ae030f5f9635e